IsraelHiking.Web/sources/application/components/sidebar/layersSidebar.css

```diff
@@ -4,8 +4,12 @@
     margin-bottom: 5px;
 }
 
-.row-hover:hover {
-    text-decoration: none;
-    color: black;
-    background: #bbb;
+.row-hover {
+    transition: background .2s ease;
 }
+
+    .row-hover:hover {
+        text-decoration: none;
+        color: black;
+        background: #bbb;
+    }
```

Comment:
Basically yes, but I didn't saw this class in others places. But of course we can move it.